tests/unit/exporters/test_console_api_error_exporter.py

```diff
@@ -0,0 +1,122 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+import json
+from unittest.mock import MagicMock
+
+import pytest
+from rich.console import Console
+
+from aiperf.exporters.console_api_error_exporter import (
+    ConsoleApiErrorExporter,
+    MaxCompletionTokensDetector,
+)
+from aiperf.exporters.exporter_config import ExporterConfig
+
+
+class MockErrorDetails:
+    def __init__(
+        self, code=400, type="Bad Request", message="", cause=None, details=None
+    ):
+        self.code = code
+        self.type = type
+        self.message = message
+        self.cause = cause
+        self.details = details
+
+
+class MockErrorDetailsCount:
+    def __init__(self, error_details, count):
+        self.error_details = error_details
+        self.count = count
+
+
+def make_summary(err):
+    return [MockErrorDetailsCount(err, 1)]
+
+
+@pytest.fixture
+def basic_error_payload():
+    return json.dumps(
+        {
+            "message": "[{'type': 'extra_forbidden','loc': ('body','max_completion_tokens'),"
+            "'msg': 'Extra inputs are not permitted'}]"
+        }
+    )
+
+
+def test_detector_detects_max_completion_tokens_error(basic_error_payload):
```

Comment:
create a TestBlahBlah class that all of these tests are under. See other test files for insight. It helps with test organization. It also allows you to run specific groups of tests easier.